### PR TITLE
[AUD-1661] Improve ProfileScreen load time

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Drastically improves track, collection, and profile screen load time by using ids from navigation context instead of waiting for ids from redux

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-aud-1661-perf-profile-screen-load

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-aud-1661-perf-profile-screen-load

File: packages/mobile/src/screens/track-screen/TrackScreen.tsx [hunk 1]
```diff
@@ -111,9 +117,27 @@ const TrackScreenMainContent = ({
...
   const lineup = useSelectorWeb(getMoreByArtistLineup, isEqual)
-  const track = useSelectorWeb(getTrack)
-  const user = useSelectorWeb(getUser)
+
+  const ownerHandle = user?.handle
+  const permalink = track?.permalink
+
+  const loadMore = useCallback(
+    (offset: number, limit: number, overwrite: boolean) => {
+      console.log(offset, limit, overwrite)
```
Comment (raymondjacobson) [inline packages/mobile/src/screens/track-screen/TrackScreen.tsx:131]: 🎁

File: packages/web/src/common/store/pages/collection/selectors.ts [hunk 2]
```diff
@@ -17,15 +17,23 @@ export const getCollectionStatus = (state: CommonState) =>
...
-export const getCollection = (state: CommonState) => {
+export const getCollection = (state: CommonState, params?: { id: ID }) => {
   const smartCollectionVariant = getSmartCollectionVariant(state)
   if (smartCollectionVariant) {
     return getSmartCollection(state, { variant: smartCollectionVariant })
   }
-  return getCachedCollection(state, { uid: getCollectionUid(state) })
+
+  const config = params?.id
+    ? { id: params?.id }
```
Comment (raymondjacobson) [inline packages/web/src/common/store/pages/collection/selectors.ts:27]: I wouldn't think you'd need params*?*.id the second time right?

Comment (dylanjeffers) [reply packages/web/src/common/store/pages/collection/selectors.ts:27]: yup

**Pull request merged**